exercises/saddle-points/test/test_saddle_points.c

```diff
@@ -44,18 +44,20 @@ static void test_single_saddle_point(void)
 
    size_t expected_count = 1;
    saddle_point_t expected_points[] = { {2, 1} };
-   saddle_points_t *actual = saddlePoints(3, 3, matrix);
+   saddle_points_t *actual = saddle_points(3, 3, matrix);
```

Comment:
Good catch on the function name 👍

Reply:
Had to go change the same for all the other exercises I've submitted recently! 🤦‍♂